test/example.test.js

```diff
@@ -115,60 +115,7 @@ test5('addProduct should take in a product object and add it to local storage (r
-            shipCategory: 'Excelsior class.',
-            shipPrice: 3,
-        },
-        {
-            id: 'enterprise-c',
-            shipName: 'Ambassador-class Enterprise-C',
-            shipImage: '../assets/Ambassador-class-Enterprise-C.jpg',
-            shipDescription: 'The Ambassador-class features the saucer section-engineering section-warp nacelle layout common to most Starfleet vessels. It is intermediate in size between the Excelsior-class and the Galaxy-class. In addition, the ship boasts at least three engineering levels for all your science needs!',
-            shipCategory: 'Ambassador class.',
-            shipPrice: 6,
-        },
-        {
-            id: 'daedalus',
-            shipName: 'USS Daedalus',
-            shipImage: '../assets/USS-Daedalus.jpg',
-            shipDescription: 'The Daedalus may be old, but you can rely on this ship, especially for the cost. The Daedalus-class was an early class of Starfleet vessel that, unlike most Federation starship designs, was designed with a spherical primary hull, outwardly similar to the later Olympic-class. This ship, which operated with a crew of approximately 229, was decommissioned by the year 2196. If you do ever find yourself and your crew stranded, it had a unique subspace distress call sure to attract friends to help you out!',
-            shipCategory: 'Antiquated space junk.',
-            shipPrice: 1,
-        },
-        {
-            id: 'honshu',
-            shipName: 'Nebula-Class USS Honshu',
-            shipImage: '../assets/Nebula-Class-USS-Honshu.jpg',
-            shipDescription: 'The Nebula-class shares a similar design lineage with its larger Galaxy-class counterpart, notably its primary and secondary hulls and nacelles. Atop the primary hull is a superstructure which can support a variety of modules, such as the inclusion of a triangular platform, fitted with torpedo launchers, an oval platform, or additional warp nacelles.',
-            shipCategory: 'Nebula class.',
-            shipPrice: 4,
-        },
-        {
-            id: 'reliant',
-            shipName: 'Miranda-class Reliant',
-            shipImage: '../assets/Miranda-class-Reliant.jpg',
-            shipDescription: 'The configuration of the Miranda-class shares a similar design lineage and features with our refitted Constitution-class. The Miranda-class is composed of a single primary hull, consisting of a saucer that is similar to that of the Constitution-class; however, the bridge module, positioned on top in the center, is shaped differently than the Constitution-class module.',
-            shipCategory: 'Miranda class.',
-            shipPrice: 5,
-        },
-        {    
-            name: 'Look at me imma ship',
-            shipPrice: 2
-        },
-    ];
+    const expectation = null;
```

Comment:
doesn't seem like your shipsFromLocalStorage is equivalent to an addProduct function. You're looking to test that your array of products is returning with a new product object added to the array.  You could then implement that function into your event listener.  I believe you'll have to do some refactoring to make, implement and test that function.